source/release-notes.txt

```diff
@@ -10,6 +10,58 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.38.0
+----------------
+
+*Released June 21, 2023*
+
+New Features:
+
+- OIDC authentication (:issue:`COMPASS-6803`).
+- Stage wizard (:issue:`COMPASS-6814`).
+- Add visual tree and update summary for the aggregation explain plan
+  (:issue:`COMPASS-6821` and :issue:`COMPASS-6888`).
+- Add types docs link to import preview (:issue:`COMPASS-6632`).
+- Open a collection in a new tab shortcut.
+- Add performance tab indicator to state that information about certain
+  collections is missing (:issue:`COMPASS-6593`).
+- Add zoom control to the modal tree view (:issue:`COMPASS-6843`).
```

Comment:
(question)
This seems fairly internal. Does it make sense to delete?

Reply:
Yup!

Reply:
Done.